RxSwiftWorkshop/Shared/SchipolService/SchipolService.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Created by Maciej Oczko on 01.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import Marshal
+import Keys
+
+protocol SchipolCallable {
+    func flights() -> Observable<[Flight]>
+}
+
+struct SchipolService: SchipolCallable {
+    struct Constants {
+        static let baseURL = "https://api.schiphol.nl/public-flights/"
```

Comment:
Maybe we could provide a URL object here and avoid the fatalError/implicit unwrap at the usage site?